src/main/resources/assets/securitycraft/lang/ko_kr.json

```diff
@@ -0,0 +1,809 @@
+	"block.securitycraft.reinforced_red_concrete": "강화된 빨강색 콘크리트",
+	"block.securitycraft.reinforced_black_concrete": "강화된 검은색 콘크리트",
+	"block.securitycraft.motion_activated_light": "모션등",
+	"block.securitycraft.reinforced_obsidian": "강화된 흑요석",
+	"block.securitycraft.reinforced_end_stone": "강화된 엔드 돌",
+	"block.securitycraft.reinforced_netherrack": "강화된 네더랙",
+	"block.securitycraft.reinforced_sea_lantern": "강화된 바다 랜턴",
+	"block.securitycraft.reinforced_bone_block": "강화된 뼈 블록",
+	"block.securitycraft.reinforced_white_carpet": "강화된 하얀색 양탄자",
+	"block.securitycraft.reinforced_orange_carpet": "강화된 주황색 양탄자",
+	"block.securitycraft.reinforced_magenta_carpet": "강화된 자홍색 양탄자",
+	"block.securitycraft.reinforced_light_blue_carpet": "강화된 하늘색 양탄자",
+	"block.securitycraft.reinforced_yellow_carpet": "강화된 노랑색 양탄자",
+	"block.securitycraft.reinforced_lime_carpet": "강화된 연두색 양탄자",
+	"block.securitycraft.reinforced_pink_carpet": "강화된 분홍색 양탄자",
+	"block.securitycraft.reinforced_gray_carpet": "강화된 회색 양탄자",
+	"block.securitycraft.reinforced_light_gray_carpet": "강화된 회백색 양탄자",
+	"block.securitycraft.reinforced_cyan_carpet": "강화된 청록색 양탄자",
+	"block.securitycraft.reinforced_purple_carpet": "강화된 보라색 양탄자",
+	"block.securitycraft.reinforced_blue_carpet": "강화된 파란색 양탄자",
+	"block.securitycraft.reinforced_brown_carpet": "강화된 갈색 양탄자",
+	"block.securitycraft.reinforced_green_carpet": "강화된 초록색 양탄자",
+	"block.securitycraft.reinforced_red_carpet": "강화된 빨강색 양탄자",
+	"block.securitycraft.reinforced_black_carpet": "강화된 검은색 양탄자",
+	"block.securitycraft.reinforced_glowstone": "강화된 발광석",
+	"block.securitycraft.gravel_mine": "자갈 지뢰",
+	"block.securitycraft.reinforced_sand": "강화된 모래",
+	"block.securitycraft.reinforced_gravel": "강화된 자갈",
+	"block.securitycraft.reinforced_stripped_oak_log": "강화된 껍질 벗긴 참나무 원목",
+	"block.securitycraft.reinforced_stripped_spruce_log": "강화된 껍질 벗긴 가문비나무 원목",
+	"block.securitycraft.reinforced_stripped_birch_log": "강화된 껍질 벗긴 자작나무 원목",
+	"block.securitycraft.reinforced_stripped_jungle_log": "강화된 껍질 벗긴 정글 나무 원목",
+	"block.securitycraft.reinforced_stripped_acacia_log": "강화된 껍질 벗긴 아카시아나무 원목",
+	"block.securitycraft.reinforced_stripped_dark_oak_log": "강화된 껍질 벗긴 짙은 참나무 원목",
+	"block.securitycraft.reinforced_stripped_oak_wood": "강화된 껍질 벗긴 참나무",
+	"block.securitycraft.reinforced_stripped_spruce_wood": "강화된 껍질 벗긴 가문비나무",
+	"block.securitycraft.reinforced_stripped_birch_wood": "강화된 껍질 벗긴 자작나무",
+	"block.securitycraft.reinforced_stripped_jungle_wood": "강화된 껍질 벗긴 정글 나무",
+	"block.securitycraft.reinforced_stripped_acacia_wood": "강화된 껍질 벗긴 아카시아나무",
+	"block.securitycraft.reinforced_stripped_dark_oak_wood": "강화된 껍질 벗긴 짙은 참나무",
+	"block.securitycraft.reinforced_oak_wood": "강화된 참나무",
+	"block.securitycraft.reinforced_spruce_wood": "강화된 가문비나무",
+	"block.securitycraft.reinforced_birch_wood": "강화된 자작나무",
+	"block.securitycraft.reinforced_jungle_wood": "강화된 정글 나무",
+	"block.securitycraft.reinforced_acacia_wood": "강화된 아카시아나무",
+	"block.securitycraft.reinforced_dark_oak_wood": "강화된 짙은 참나무,
```

Comment:
There's a missing closing quotation mark at the end of this line.

Reply:
I fixed.